src/interaction/FixedPairListTypesInteractionTemplate.hpp

```diff
@@ -128,31 +130,91 @@ inline void FixedPairListTypesInteractionTemplate<_Potential>::addForces()
 {
     LOG4ESPP_INFO(theLogger, "add forces computed by the FixedPair List");
     const bc::BC &bc = *getSystemRef().bc;
+    real offs = getSystemRef().shearOffset;
+    bool shear_flag = (getSystemRef().ifShear && offs != .0);
 
-    for (FixedPairList::PairList::Iterator it(*fixedpairList); it.isValid(); ++it)
+    if (shear_flag)
     {
-        Particle &p1 = *it->first;
-        Particle &p2 = *it->second;
-        int type1 = p1.type();
-        int type2 = p2.type();
-        Potential &potential = getPotential(type1, type2);
-        // std::shared_ptr<Potential> potential = getPotential(type1, type2);
-
-        Real3D force(0.0);
-        // if(potential._computeForce(force, p1, p2)) {
-        ////if(potential->_computeForce(force, p1, p2)) {
-        //  p1.force() += force;
-        //  p2.force() -= force;
-        //  LOG4ESPP_TRACE(theLogger, "id1=" << p1.id() << " id2=" << p2.id() << " force=" <<
-        //  force);
-        //}
-        Real3D dist;
-        bc.getMinimumImageVectorBox(dist, p1.position(), p2.position());
-        potential.computeColVarWeights(dist, bc);
-        if (potential._computeForce(force, p1, p2, dist))
+        real Lx = bc.getBoxL()[0];
+        real Lz = bc.getBoxL()[2];
+
+        for (FixedPairList::PairList::Iterator it(*fixedpairList); it.isValid(); ++it)
+        {
+            Particle &p1 = *it->first;
+            Particle &p2 = *it->second;
+            int type1 = p1.type();
+            int type2 = p2.type();
+            Potential &potential = getPotential(type1, type2);
+            // shared_ptr<Potential> potential = getPotential(type1, type2);
+
+            Real3D force(0.0);
+            // if(potential._computeForce(force, p1, p2)) {
+            ////if(potential->_computeForce(force, p1, p2)) {
+            //  p1.force() += force;
+            //  p2.force() -= force;
+            //  LOG4ESPP_TRACE(theLogger, "id1=" << p1.id() << " id2=" << p2.id() << " force=" <<
+            //  force);
+            //}
```

Comment:
Is it important?

Reply:
but it also appears in the master branch
anyway, let me move the log_trace to ln 182 and remove the rest